core/spec.md

```diff
@@ -2101,8 +2101,11 @@ The following describes the attributes of Registry model:
   - Values MAY be empty strings.
   - The following labels are defined for convenience and MAY also be used
     as labels for Groups or Resource model types:
+    - Name: `modelversion`
+      Value: The version of the model. For many specs its a combination of
+      major and minor, e.g. `1.2`.
     - Name: `compatiblewith`
-      Value: A URI reference to Registry that this Registry's model is
+      Value: An absolute URL to the model specification that this model is
```

Comment:
@duglin suggestion for different phrasing, using the new URL terms. I think this has to be absolute URL.

Reply:
ok I'll pick this up for the other PR- then we don't need to worry about rebasing and you can drop this file from your PR.

Reply:
I've squashed it now as you suggested